### PR TITLE
Feature/update acm and add external apps

### PR DESCRIPTION
**Pull request by cecemel**
Two things covered in the PR:
- update to the new acm context for databankErediensten
-  added access to new module: Eredienst organisaties (same pattern as databankErediensten)

please feel free to correct immediately when it burns

### CODE REVIEW

File: app/templates/index.hbs [hunk 1]
```diff
@@ -171,16 +171,36 @@
...
       {{/if}}
+      {{#if this.currentSession.canAccessWorshipOrganisationsDb}}
+       <li class="au-o-grid__item au-u-1-2@medium au-u-1-3@large">
+           <LoketModuleCard
+             @icon="link-external"
+             @userManualLink="https://abb-vlaanderen.gitbook.io/handleiding-loket/erediensten/eredienst-organisaties"
+           >
+             <:title>Eredienst Organisaties</:title>
+             <:description>Consulteer hier de mandatarissen en bedienaren
+             van de besturen eredienst en centrale besturen. </:description>
```
Comment (Windvis) [inline app/templates/index.hbs:192]: Current version sounds a bit weird, not sure if this is better:
```suggestion
             van de eredienst besturen en centrale besturen. </:description>
```

File: app/templates/index.hbs [hunk 2]
```diff
@@ -171,16 +171,36 @@
...
+             <:link>
+             <AuLinkExternal
+               @skin="button"
+               href={{(worship-organisations-database-url)}}
+             >
+               Ga naar de eredienst organisaties (externe link)
```
Comment (Windvis) [inline app/templates/index.hbs:198]: Same here
```suggestion
               Ga naar de eredienst organisaties databank (externe link)
```

**Review by Windvis — COMMENTED** [review]
Some small copy remarks but should be good to go otherwise 👍.

File: app/templates/index.hbs (hunk 2 above)
Comment (cecemel) [reply app/templates/index.hbs:198]: I don't think it's called databank in the end...

Comment (Windvis) [reply app/templates/index.hbs:198]: Yea, I didn't know about the issue. I assumed it was another DB app 😄.

**Review by Windvis — APPROVED** [review]

**Pull request merged**